sdk/CHANGELOG.md

```diff
@@ -27,6 +27,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `Account::{unspent_alias_output, unspent_foundry_output, unspent_nft_output}` methods;
 - `StrongholdAdapter::inner` method;
 
+### Changed
+
+- `StorageManager` now accepts a generic storage instead of a boxed type, and the params have been reversed;
```

Comment:
Not a big deal but technically not needed as it's not public

Reply:
Looks pretty public to me, but maybe I'm missing something.

Reply:
`StorageManager::new` is `pub(crate)` ?

Reply:
😆 I completely missed that